packages/eas-cli/src/prompts.ts

```diff
@@ -55,3 +55,18 @@ export async function toggleConfirmAsync(
   );
   return value ?? null;
 }
+
+export async function pressAnyKeyToContinueAsync() {
+  process.stdin.setRawMode(true);
+  process.stdin.resume();
+  process.stdin.setEncoding('utf8');
+
+  await new Promise(res => {
+    process.stdin.on('data', key => {
+      if (String(key) === '\u0003') {
```

Comment:
Why are we checking for `END OF TEXT` (https://www.fileformat.info/info/unicode/char/0003/index.htm) here?

Reply:
that is a code for `ctrl-c`
there is a comment line bellow